anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -154,14 +165,18 @@ def toggle_feed_enabled(feed, enabled):
 def toggle_group_enabled(feed, group, enabled):
     if type(enabled) != bool:
         raise BadRequest(message="state must be a boolean", detail={"value": enabled})
-    if feed == GRYPE_DB_FEED_NAME:
+    provider = get_vulnerabilities_provider()
+    internal_feed_name = provider.display_mapper.get_internal_name(feed)
+    if isinstance(provider, GrypeProvider) and internal_feed_name == GRYPE_DB_FEED_NAME:
         raise HTTPNotImplementedError(
-            message="Enabling and disabling groups for grypedb feed is not currently supported.",
+            message="Enabling and disabling groups for {} feed with the grype vulnerability provider enabled is not currently supported.".format(
+                feed
+            ),
             detail={},
         )
     session = db.get_session()
     try:
-        g = db.set_feed_group_enabled(session, feed, group, enabled)
+        g = db.set_feed_group_enabled(session, internal_feed_name, group, enabled)
```

Comment:
I know this was already there, but can you rename the single-character variable?

Reply:
done